projects/demo/src/modules/directives/copy-processor/copy-processor.template.html

```diff
@@ -0,0 +1,45 @@
+<tui-doc-page
+    i18n-header
+    header="CopyProcessor"
+    package="KIT"
+    type="directives"
+>
+    <ng-template pageTab>
+        <p i18n>Directive is used to processed text via coping</p>
```

Comment:
"when" instead of "via" makes more sense :)

Reply:
done